modules/auth/assembly-enabling-authentication.adoc

```diff
@@ -0,0 +1,34 @@
+[id='enabling-authentication']
+= Enabling authentication in {product}
+
+Authentication within {product} facilitates user sign-in, identification, and access to external resources.
+It supports multiple authentication providers.
+
+Authentication providers are typically used in the following ways:
+
+- One provider for sign-in and identification.
+- Additional providers for accessing external resources.
+
+The {product} supports the following authentication providers:
+
+Auth0:: `auth0`
+Atlassian:: `atlassian`
+Microsoft Azure:: `microsoft`
+Microsoft Azure Easy Auth:: `azure-easyauth`
+Bitbucket:: `bitbucket`
+Bitbucket Server:: `bitbucketServer`
+Cloudflare Access:: `cfaccess`
+GitHub:: `github`
+GitLab:: `gitlab`
+Google:: `google`
+Google IAP:: `gcp-iap`
+OIDC:: `oidc`
+Okta:: `okta`
+OAuth 2 Custom Proxy:: `oauth2Proxy`
+OneLogin:: `onelogin`
+SAML:: `saml`
+
+For each provider, follow the dedicated procedure to:
```

Comment:
```suggestion
For each provider, follow the dedicated procedure to:
```

I assume the user is expected to either google the provider's config details, or CTRL-F in the admin guide for details?

Reply:
I was expecting that the list above will have links to configuration page (section) for each provider. Is that right @themr0c ?

Reply:
Yes, when the procedures for each provider will follow, it will make more sense

Reply:
https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/78 will come first, then the others, when we have settled on a structure that satisfies everybody.